src/Acuminator/Acuminator.Analyzers/Acuminator.Analyzers.csproj

```diff
@@ -8,19 +8,19 @@
     <Copyright>Copyright © 2017-2019 Acumatica Ltd.</Copyright>
     <LangVersion>8.0</LangVersion>
     <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>..\..\key.snk</AssemblyOriginatorKeyFile>
```

Comment:
The type of the key should not be changed, since our build infrastructure and all other participants use .snk key